### PR TITLE
Feature/scale bar

### LINKED ISSUE #15: Load assets (SVG)
We need to load in static assets like SVGs to use in the UI.

Acceptance criteria:
Rendering an imported SVG in the UI.

### PR DESCRIPTION
**Pull request by interim17**
Closes #15 and #19

SVG assets, what a journey. Found a complicated workaround to load them as separate files, use two loaders, touch every config file, and converting the strings to URIs, then still found they didn't want to be styled!

But you know what does work apparently? Just putting declaring variables as `const icon = <svg>.... </svg>` in a `tsx` file.

Simpler, it works. Feels too easy after all this fuss, am I breaking a law or something? I don't love having the all the `svg` guts just sitting in a component file but its better than the alternative?

## Scalebar:
Computed and rendered almost identically to the simularium website repo, in a container with camera controls that will also hold the forthcoming playback controls. Computing the label in `Viewer` and passing it down, but open to feedback/strategy on what to hold in Viewer as features pile up and that file gets top heavy.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -7848,15 +7876,19 @@ __metadata:
...
     jest: ^29.7.0
     jest-cli: ^29.7.0
     jest-environment-jsdom: ^29.5.0
+    jest-transform-stub: ^2.0.0
     jsdom-global: ^3.0.2
+    mini-svg-data-uri: ^1.4.4
     mkdirp: ^0.5.1
     npm-run-all: ^4.1.3
     prettier: ^2.0.5
+    raw-loader: ^4.0.2
```
Comment (ascibisz) [inline yarn.lock]: If I don't include raw-loader, I get the following compilation error. We aren't specifically using raw-loader, but it seems like jupyter notebook is somewhere / somehow?

```
Error: resolve 'raw-loader' in '/Users/alli.scibisz/nbsv'
  Parsed request is a module
  using description file: /Users/alli.scibisz/nbsv/package.json (relative path: .)
    resolve as module
      looking for modules in /Users/alli.scibisz/nbsv/node_modules
        single file module
          using description file: /Users/alli.scibisz/nbsv/package.json (relative path: ./node_modules/raw-loader)
            no extension
              /Users/alli.scibisz/nbsv/node_modules/raw-loader doesn't exist
            .js
              /Users/alli.scibisz/nbsv/node_modules/raw-loader.js doesn't exist
        /Users/alli.scibisz/nbsv/node_modules/raw-loader doesn't exist
      /Users/alli.scibisz/node_modules doesn't exist or is not a directory
      /Users/node_modules doesn't exist or is not a directory
      /node_modules doesn't exist or is not a directory
    at compilerCallback (/Users/alli.scibisz/nbsv/node_modules/@jupyterlab/builder/lib/build-labextension.js:100:19)
    at /Users/alli.scibisz/nbsv/node_modules/@jupyterlab/builder/lib/build-labextension.js:142:21
    at /Users/alli.scibisz/nbsv/node_modules/neo-async/async.js:2830:7
    at done (/Users/alli.scibisz/nbsv/node_modules/neo-async/async.js:2865:11)
    at /Users/alli.scibisz/nbsv/node_modules/neo-async/async.js:2818:7
    at /Users/alli.scibisz/nbsv/node_modules/@jupyterlab/builder/node_modules/webpack/lib/HookWebpackError.js:68:3
    at Hook.eval [as callAsync] (eval at create (/Users/alli.scibisz/nbsv/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:15:1)
    at Hook.CALL_ASYNC_DELEGATE [as _callAsync] (/Users/alli.scibisz/nbsv/node_modules/tapable/lib/Hook.js:18:14)
    at Cache.shutdown (/Users/alli.scibisz/nbsv/node_modules/@jupyterlab/builder/node_modules/webpack/lib/Cache.js:150:23)
    at /Users/alli.scibisz/nbsv/node_modules/@jupyterlab/builder/node_modules/webpack/lib/Compiler.js:1229:15
```

File: examples/introduction.ipynb [hunk 2]
```diff
@@ -9,7 +9,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 9,
```
Comment (ShrimpCryptid) [inline examples/introduction.ipynb:12]: Tiniest nit: I feel like it would be nice to reset these execution counts 😅

File: src/Viewer.tsx [hunk 3]
```diff
@@ -1,45 +1,48 @@
+import React, { useEffect, useRef, useState } from 'react';
 import SimulariumViewer, {
   RenderStyle,
   SimulariumController,
+  TrajectoryFileInfo,
 } from '@aics/simularium-viewer';
-import React, { useEffect, useRef, useState } from 'react';
+import { ModelInfo } from '@aics/simularium-viewer/type-declarations/simularium/types';
 
 import { WidgetModel } from '@jupyter-widgets/base';
```
Comment (ShrimpCryptid) [inline src/Viewer.tsx]: Nit: Group jupyter import w/ other external imports

File: src/components/ScaleBar.tsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+
+import { ScaleBarIcon } from './Icons';
+
+import '../../css/scale_bar.css';
```
Comment (ShrimpCryptid) [inline src/components/ScaleBar.tsx:5]: This is making me remember that conversation we were having about styled-components. I can definitely see the use case here!

**Review by ShrimpCryptid — APPROVED** [review]
Looks good, was able to build and test locally without issues.

File: src/components/ScaleBar.tsx (hunk 4 above)
Comment (interim17) [reply src/components/ScaleBar.tsx:5]: Yeah feeling some time pressure to get MVP done with the way I'm used to developing, but want to move into that CSS-in-JS approach soon, maybe if there is a conversation around bumping the antd version in sim-website at some point.

**Review by meganrm — APPROVED** [review]

**Pull request merged**